app/src/control_msg.h

```diff
@@ -25,6 +25,7 @@ enum control_msg_type {
     CONTROL_MSG_TYPE_GET_CLIPBOARD,
     CONTROL_MSG_TYPE_SET_CLIPBOARD,
     CONTROL_MSG_TYPE_SET_SCREEN_POWER_MODE,
+    CONTROL_MSG_TYPE_TURN_SCREEN_OFF,
```

Comment:
We need to distinguish between "turn screen off but keep device on" (like [here](https://github.com/Genymobile/scrcpy/blob/513d1ac96d24d6cf07fb1850232b5c0e8f555615/app/src/scrcpy.c#L391)) and "turn screen off and device off (standby)". (There is already some naming confusion in the current codebase, if you have any suggestion…)

Reply:
Maybe lock screen vs turn screen off?